config/src/main/java/org/springframework/security/config/annotation/web/reactive/ServerHttpSecurityConfiguration.java

```diff
@@ -113,6 +119,10 @@ public CurrentSecurityContextArgumentResolver reactiveCurrentSecurityContextArgu
 
 	@Bean(HTTPSECURITY_BEAN_NAME)
 	@Scope("prototype")
+	public ServerHttpSecurity serverHttpSecurity() {
```

Comment:
There is a test that uses the `httpSecurity()` method. 
https://github.com/spring-projects/spring-security/blob/047bd16b5156de0bb595ff6b324a63cef92b1fe1/config/src/test/java/org/springframework/security/config/annotation/web/reactive/ServerHttpSecurityConfigurationBuilder.java#L31
If we separate the bean method from the method that creates the object then we avoid calling the bean method in the test.
Unless we aren't concerned about the test using the bean method because it's creating a new instance of `ServerHttpSecurityConfiguration`?

Reply:
When it just performs new it won't create a Bean since nothing is processing the annotation.